phoenicis-configuration/src/main/java/org/phoenicis/configuration/localisation/Localisation.java

```diff
@@ -50,28 +50,37 @@ public static void setAdditionalTranslations(ResourceBundle resourceBundle) {
         getI18n().setResources(resourceBundle);
     }
 
-    public static String tr(String str) {
-        return getI18n().tr(str);
+    public static String tr(String text) {
+        return text == null ? "" : getI18n().tr(text);
     }
 
     public static String tr(String text, Object o1) {
-        return getI18n().tr(text, o1);
+        return text == null ? "" : getI18n().tr(text, o1);
     }
 
     public static String tr(String text, Object o1, Object o2) {
-        return getI18n().tr(text, o1, o2);
+        return text == null ? "" : getI18n().tr(text, o1, o2);
     }
 
     public static String tr(String text, Object o1, Object o2, Object o3) {
-        return getI18n().tr(text, o1, o2, o3);
+        return text == null ? "" : getI18n().tr(text, o1, o2, o3);
```

Comment:
If you use "null" as a translation for the not existing value `null` in the gui you will have "null"s in your gui components. I'm not sure we want that.